packages/runtime/src/runtime/menu.tsx

```diff
@@ -165,12 +166,13 @@ export const EnsembleMenu: React.FC<{
   const [isCollapsed, setIsCollapsed] = useState<boolean>(
     type === EnsembleMenuModelType.Drawer,
   );
-
+  const [defaultOpenKeys, setDefaultOpenKeys] = useState<string>();
```

Comment:
@sagardspeed2 shouldn't `defaultOpenKeys` be an array as the name suggests and antD is also expecting an array